package.json

```diff
@@ -12,9 +12,9 @@
   ],
   "main": "index.js",
   "scripts": {
-    "lint": "eslint .",
+    "lint": "eslint . --ignore-path .gitignore",
```

Comment:
I replied to the comment at https://github.com/strongloop/eslint-config-loopback/pull/11#issuecomment-245736649

Anyways, it's just futureproofing as the files you to `.gitignore` usually don't need linting either (and if they do, they are not checked in anyways).

However, I will revert this until you comment further on the link above.